src/index-sdk/gmiStaking.ts

```diff
@@ -136,5 +136,26 @@ export const getAmountOfStakedTokens = async (
   return await contract.methods.totalSupply().call()
 }
 
+export const getApy = async (
```

Comment:
It seems like the user will only be able to view their APY once they deposit here. Is that correct?
If so, shouldn't there be a way to show APY without the user needing to deposit?

Reply:
Yeah would be much better to show APR before user deposits. You should be able to calculate APR without tapping into User balances. https://discord.com/channels/762061559744299010/875052017591128084/930223193187303495